integrations/cloudbuild/build-coverage-remote.yaml

```diff
@@ -0,0 +1,23 @@
+steps:
+    - name: gcr.io/$PROJECT_ID/remote-builder
+      env:
+          - GIT_CLONE_CMD=git clone --recurse-submodules https://github.com/project-chip/connectedhomeip.git;
```

Comment:
Lets run each comand independetly: we can clone and run coverage in different images in cloudbuild. I believe the git image is small enough to not be too expensive.

`/workspace` should be shared among runs and if we need more we can use volumes

Reply:
I see, this connects to a VM.

In that case, why do we need to pass in all these commands inside yaml? We hardcode sequence and logic inside the run script anyway. May as well set the checkout and commands inside that script.